01week/rockPaperScissors.js

```diff
@@ -8,10 +8,28 @@ const rl = readline.createInterface({
 });
 
 
-function rockPaperScissors(hand1, hand2) {
+function rockPaperScissors(arg1, arg2) {
+  
+  const hand1 = arg1.trim().toLowerCase();
+  const hand2 = arg2.trim().toLowerCase();
+  //console.log(hand1,hand2)
```

Comment:
Try not leave any console logs in your code.  These are great for testing your code, but takes up space.

Reply:
I fixed this.